src/cli/definitions/parse-server.js

```diff
@@ -195,6 +195,11 @@ export default {
     help: "The TTL for caching the schema for optimizing read/write operations. You should put a long TTL when your DB is in production. default to 0; disabled.",
     action: numberParser("schemaCacheTTL"),
   },
+  "schemaCacheFrozen": {
```

Comment:
probably name to freezeSchema as, yes we leverage the cache for the freezing, but we want it to do it at large.